serpextract/serpextract.py

```diff
@@ -614,7 +637,6 @@ def extract(serp_url, parser=None, lower_case=True, trimmed=True,
             query = _unicode_parse_qs(url_parts.query, keep_blank_values=True)
             for param in _naive_params:
                 if param in query:
-                    import tldextract
```

Comment:
Oh yeah, I also moved this import to the top level because repeated imports are insanely slow.